### ISSUE TITLE
Flux 工作流中XY比较图表报错

### ISSUE THREAD

**Issue by terliu** [yolain/ComfyUI-Easy-Use#340]
如下图：
![image](https://github.com/user-attachments/assets/7d4fb0b2-92cc-4a82-af55-49eb22a51804)
 错误如下：
Error occurred when executing easy kSampler:

mat1 and mat2 shapes cannot be multiplied (7488x16 and 64x3072)

File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\execution.py", line 317, in execute
output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\execution.py", line 192, in get_output_data
return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\execution.py", line 169, in _map_node_over_list
process_inputs(input_dict, i)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\execution.py", line 158, in process_inputs
results.append(getattr(obj, func)(**inputs))
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use\py\easyNodes.py", line 5414, in simple
return super().run(pipe, None, None, None, None, None, image_output, link_id, save_prefix,
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use\py\easyNodes.py", line 5381, in run
return process_xyPlot(pipe, samp_model, samp_clip, samp_samples, samp_vae, samp_seed, samp_positive, samp_negative, steps, cfg, sampler_name, scheduler, denoise, image_output, link_id, save_prefix, tile_size, prompt, extra_pnginfo, my_unique_id, preview_latent, xyPlot, force_full_denoise, disable_noise, samp_custom)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use\py\easyNodes.py", line 5323, in process_xyPlot
latents_plot = sampleXYplot.get_labels_and_sample(plot_image_vars, latent_image, preview_latent, start_step,
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use\py\libs\xyplot.py", line 523, in get_labels_and_sample
self.image_list, self.max_width, self.max_height, self.latents_plot = self.sample_plot_image(
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use\py\libs\xyplot.py", line 448, in sample_plot_image
samples = self.sampler.common_ksampler(model, seed, steps, cfg, sampler_name, scheduler, positive, negative, samples,
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use\py\libs\sampler.py", line 115, in common_ksampler
samples = comfy.sample.sample(model, noise, steps, cfg, sampler_name, scheduler, positive, negative,
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\Comfyui-StableSR\nodes.py", line 69, in hook_sample
return original_sample(*args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\sample_error_enhancer.py", line 22, in informative_sample
raise e
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\sample_error_enhancer.py", line 9, in informative_sample
return original_sample(*args, **kwargs) # This code helps interpret error messages that occur within exceptions but does not have any impact on other operations.
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-AnimateLCM\animatediff\sampling.py", line 241, in motion_sample
return orig_comfy_sample(model, noise, *args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-AnimateDiff-Evolved\animatediff\sampling.py", line 434, in motion_sample
return orig_comfy_sample(model, noise, *args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Advanced-ControlNet\adv_control\sampling.py", line 116, in acn_sample
return orig_comfy_sample(model, *args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Advanced-ControlNet\adv_control\utils.py", line 116, in uncond_multiplier_check_cn_sample
return orig_comfy_sample(model, *args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\sample.py", line 43, in sample
samples = sampler.sample(noise, positive, negative, cfg=cfg, latent_image=latent_image, start_step=start_step, last_step=last_step, force_full_denoise=force_full_denoise, denoise_mask=noise_mask, sigmas=sigmas, callback=callback, disable_pbar=disable_pbar, seed=seed)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_smZNodes\smZNodes.py", line 1447, in KSampler_sample
return _KSampler_sample(*args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\samplers.py", line 829, in sample
return sample(self.model, noise, positive, negative, cfg, self.device, sampler, sigmas, self.model_options, latent_image=latent_image, denoise_mask=denoise_mask, callback=callback, disable_pbar=disable_pbar, seed=seed)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_smZNodes\smZNodes.py", line 1470, in sample
return _sample(*args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\samplers.py", line 729, in sample
return cfg_guider.sample(noise, latent_image, sampler, sigmas, denoise_mask, callback, disable_pbar, seed)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\samplers.py", line 716, in sample
output = self.inner_sample(noise, latent_image, device, sampler, sigmas, denoise_mask, callback, disable_pbar, seed)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\samplers.py", line 695, in inner_sample
samples = sampler.sample(self, sigmas, extra_args, callback, noise, latent_image, denoise_mask, disable_pbar)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\samplers.py", line 600, in sample
samples = self.sampler_function(model_k, noise, sigmas, extra_args=extra_args, callback=k_callback, disable=disable_pbar, **self.extra_options)
File "G:\comfyui\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
return func(*args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\k_diffusion\sampling.py", line 144, in sample_euler
denoised = model(x, sigma_hat * s_in, **extra_args)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\samplers.py", line 299, in __call__
out = self.inner_model(x, sigma, model_options=model_options, seed=seed)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_smZNodes\smZNodes.py", line 993, in __call__
return self.predict_noise(*args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_smZNodes\smZNodes.py", line 1043, in predict_noise
out = super().predict_noise(*args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\samplers.py", line 685, in predict_noise
return sampling_function(self.inner_model, x, timestep, self.conds.get("negative", None), self.conds.get("positive", None), self.cfg, model_options=model_options, seed=seed)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\samplers.py", line 279, in sampling_function
out = calc_cond_batch(model, conds, x, timestep, model_options)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\samplers.py", line 228, in calc_cond_batch
output = model.apply_model(input_x, timestep_, **c).chunk(batch_chunks)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Advanced-ControlNet\adv_control\utils.py", line 68, in apply_model_uncond_cleanup_wrapper
return orig_apply_model(self, *args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\model_base.py", line 142, in apply_model
model_output = self.diffusion_model(xc, t, context=context, control=control, transformer_options=transformer_options, **extra_conds).float()
File "G:\comfyui\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\nn\modules\module.py", line 1511, in _wrapped_call_impl
return self._call_impl(*args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\nn\modules\module.py", line 1520, in _call_impl
return forward_call(*args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\ldm\flux\model.py", line 159, in forward
out = self.forward_orig(img, img_ids, context, txt_ids, timestep, y, guidance, control)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\ldm\flux\model.py", line 104, in forward_orig
img = self.img_in(img)
File "G:\comfyui\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\nn\modules\module.py", line 1511, in _wrapped_call_impl
return self._call_impl(*args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\nn\modules\module.py", line 1520, in _call_impl
return forward_call(*args, **kwargs)
File "G:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-GGUF\ops.py", line 147, in forward
x = torch.nn.functional.linear(x, weight, bias)
![image](https://github.com/user-attachments/assets/fd444351-7aa4-4cd3-9065-b72056587619)

**Comment by yolain** [comment]
xy没有支持自定义采样器呢。

**Comment by terliu** [comment]
换成你工作流集中的采样器也不行呀：
这是你的工作流实例：
![image](https://github.com/user-attachments/assets/0ab76a3b-e1e2-49e8-aee8-d1c32b21139c)
我换成你实例的也报错
![image](https://github.com/user-attachments/assets/14bebdcf-e154-454b-94c1-ef5c95a8d7d1)
![image](https://github.com/user-attachments/assets/b175ab64-e57a-44ec-b01d-3f77b229357d)

**Comment by magician-lzf** [comment]
我用图生图的工作流就行，用文生图的工作流程跟你一样报错

**Comment by yolain** [comment]
xy的没适配呢，很麻烦。 官配的flux只能走自定义采样器

**Comment by Dragoy** [comment]
有人找到临时替代品了吗？

**Comment by turkyden** [comment]
还没兼容到 fp8 的 flux 模型

**Comment by yolain** [comment]
我更新修复下，之前官方加了修复潜空间通道后就可以 忘了去改过来了。 需要设置CFG为1
